command/src/main/kotlin/com/amarcolini/joos/hardware/drive/DiffSwerveDrive.kt

```diff
@@ -97,11 +94,11 @@ open class DiffSwerveDrive(
     }
 
     override fun setDrivePower(drivePower: Pose2d) {
-        targetSpeeds = DiffSwerveKinematics.robotToWheelVelocities(drivePower, 1.0).map {
+        targetSpeeds = DiffSwerveKinematics.robotToWheelVelocities(drivePower, constraints.trackWidth).map {
```

Comment:
If you look at `setDrivePower` in other drives, 1.0 is used in place of the actual track width.